data/registry_model/quota_registry_model.py

```diff
@@ -0,0 +1,48 @@
+from data.model import organization, repository, notification, namespacequota
```

Comment:
Just a nit, but can we move this to `data/model/quota.py`, `registry_model/` should contain only registry specific ops.